core/domain/collection_services.py

```diff
@@ -574,6 +576,29 @@ def get_collection_snapshots_metadata(collection_id):
         collection_id, version_nums)
 
 
+def _get_contributor_ids_for_collection(col_id):
```

Comment:
Alright. What priority do you think is in place here? Finish the map reduce job for impact (I'm very close to opening a PR-- in fact I might do so right now to see what other tests you think are necessary, beginning to wonder if some of the tests outlined in the design doc are in fact redundant) and then update the collection summary contributors? I don't think the update for that should be *too* difficult (famous last words...)...